spec/models/profile_model_spec.rb

```diff
@@ -89,4 +89,66 @@
       end
     end
   end
+
+  describe "ClassMethods" do
+    before :all do
+      DatabaseCleaner.start
```

Comment:
I think we can have this set up globally in the spec helper? that will ensure it's available for all specs.